packages/liveblocks-client/src/fancy-console.ts

```diff
@@ -0,0 +1,35 @@
+const badge =
+  "background:radial-gradient(106.94% 108.33% at -10% -5%,#ff1aa3 0,#ff881a 100%);border-radius:6px;color:#fff;padding:2px 6px";
+const bold = "font-weight:600";
```

Comment:
@pierrelevaillant If we want to change the style of this branded badge, we can simply change the CSS here in one place.

Reply:
I quite like this styling:
```suggestion
const badge =
  "background:radial-gradient(106.94% 108.33% at -10% -5%,#ff1aa3 0,#ff881a 100%);border-radius:3px;color:#fff;padding:2px 5px;font-family:sans-serif;font-weight:600";
const bold = "font-weight:600";
```

Try it out by pasting this in your console:
```
console.log("%cLiveblocks%c Connection to websocket...", "background:radial-gradient(106.94% 108.33% at -10% -5%,#ff1aa3 0,#ff881a 100%);border-radius:3px;color:#fff;padding:2px 5px;font-family:sans-serif;font-weight:600", "")
```

Reply:
Nice! Will use that 👍